public/data/components/config.json

```diff
@@ -39,5 +39,19 @@
         }
       }
     }
+  },
+  "transports": {
+    "defaultDistance": {
+      "air": 10000,
+      "road": null,
+      "sea": 18000
+    },
+    "modeProcesses": {
+      "boat": "20a62b2c-a543-5076-83aa-c5b7d340206a",
+      "boatCooling": "3cb99d44-24f6-5f6e-a8f8-f754fe44d641",
+      "lorry": "46e96f29-9ca5-5475-bb3c-6397f43b7a5b",
+      "lorryCooling": "219b986c-9751-58cf-977e-7ba8f0b4ae2b",
+      "plane": "3364ef93-b936-531a-8f5a-432349aef398"
```

Comment:
@n1k0 I’m working on https://github.com/MTES-MCT/ecobalyse/issues/1578 atm, so I guess the ids are ok for boat and lorry, but for plane you should probably use 326369d9-792a-5ab5-8276-c54108c80cb1
(I’ll update the well-knowns too)